test/OpenTelemetry.Tests.Stress/StressTestNativeMethods.cs

```diff
@@ -23,6 +23,7 @@ public static ulong GetCpuCycles()
     }
 
     [DllImport("kernel32.dll")]
+    [DefaultDllImportSearchPaths(DllImportSearchPath.System32)]
```

Comment:
just curious, what problem was this addressing?

Reply:
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca5392#rule-description

Reply:
Performance of invocation as I understand.